web/tutorial-actions-executor/src/tutorialApp.ts

```diff
@@ -8,24 +8,35 @@ export type AppParentDirPath = Branded<string, "AppParentDirPath">;
 export type TutorialDirPath = Branded<string, "TutorialDirPath">;
 export type PatchesDirPath = Branded<string, "PatchesDirPath">;
 
-const tutorialAppName = "TodoApp" as AppName;
-const tutorialAppParentDirPath = path.resolve("./.result") as AppParentDirPath;
-const tutorialAppDirPath = path.resolve(
-  tutorialAppParentDirPath,
-  tutorialAppName,
-) as AppDirPath;
-const docsTutorialDirPath = path.resolve("../docs/tutorial") as TutorialDirPath;
-const docsTutorialPatchesPath = path.resolve(
-  docsTutorialDirPath,
-  "patches",
-) as PatchesDirPath;
+export interface TutorialApp {
+  name: AppName;
+  parentDirPath: AppParentDirPath;
```

Comment:
Cant' we just get parent dir from dirPath? This is duplication that can potentially be incorrect, while we can easily extract that info, no?